CHANGELOG.rst

```diff
@@ -17,6 +17,9 @@ Unreleased
 
 **Changed**
 
+* Property `compas_fab.robots.Robot.artist` does not try to scale robot
```

Comment:
Maybe this also deserves a :class: prefix...

Reply:
It works nicely on the docs website but not in the Github preview, see [my fork](https://github.com/tetov/compas_fab/blob/allow_skeleton_robot_instances/CHANGELOG.rst).